src/pipx/main.py

```diff
@@ -648,14 +713,16 @@ def cli() -> int:
         if not parsed_pipx_args.command:
             parser.print_help()
             return 1
-        return run_pipx_command(parsed_pipx_args)
+        # return exit code of command or 0 if none provided
+        return run_pipx_command(parsed_pipx_args) or 0
```

Comment:
Added this because I ran into issues with `assert <some install command>` as it would sometimes return `None`.